importers/src/main/groovy/whelk/importer/WhelkCopier.groovy

```diff
@@ -82,8 +82,17 @@ class WhelkCopier {
     void save(doc) {
         System.err.println "[$copied] Copying $doc.shortId from $source.baseUri to $dest.baseUri"
 
-        def newDataRepr = doc.dataAsString.replaceAll(/"${source.baseUri}/,
-                '"'+dest.baseUri.toString())
+        def libUriPlaceholder = "___TEMP_HARDCODED_LIB_BASEURI"
+        def newDataRepr = doc.dataAsString.replaceAll( // Move all lib uris, to a temporary placeholder.
+                source.baseUri.resolve("library/").toString().replace(".", "\\."),
```

Comment:
Did you intend to remove the leading `"` here? It was added to ensure that only values starting with the baseUri are affected.

It's good to escape the dot (not doing so was just sloppy, albeit practically OK). But be more safe, surround the baseUri with `\Q` and `\E` (e.g. `/"\Q${source.baseUri}\E/`, which is what `java.util.regex.Pattern.quote` does).

Also note that the replacement string is a regular string, so the only thing ever needing an escape there is `$` (which is what `java.util.regex.Matcher.quoteReplacement` does).

Reply:
I pushed a new version, see if you like it better!